.github/workflows/publish-js-sdks.yml

```diff
@@ -17,20 +17,26 @@ jobs:
           node-version-file: "package.json"
           registry-url: "https://registry.npmjs.org"
 
+      - uses: pnpm/action-setup@v4
+        name: Install pnpm
+        with:
+          run_install: true
+
       - name: Install dependencies
-        run: npm install
-        working-directory: sdk/js/
+        run: pnpm --filter @pythnetwork/express-relay-js install
 
       - name: Run tests
-        run: npm test # Ensure your tests pass before publishing
-        working-directory: sdk/js/
+        run: pnpm --filter @pythnetwork/express-relay-js test # Ensure your tests pass before publishing
+
+      - name: Set publishing config
+        run: pnpm config set '//registry.npmjs.org/:_authToken' "${NODE_AUTH_TOKEN}"
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
 
       - name: Build package
-        run: npm run build # If your package requires a build step
-        working-directory: sdk/js/
+        run: pnpm --filter @pythnetwork/express-relay-js build # If your package requires a build step
```

Comment:
Same thing, I'd just run `pnpm -r build` here and `pnpm -r publish`, so you can have new packages hook into this CI workflow just by defining the expected lifecycle hooks

Reply:
I want to exactly publish the `@pythnetwork/express-relay-js` pacakge. Why should I build and publish all of them then?